### PR TITLE
Oak

### PR DESCRIPTION
**Pull request by SBuchness**

### CODE REVIEW

File: spencer-buchness.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<html>
+<head>
+<title>CSS Timeline</title>
+<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+<link rel="stylesheet" type="/spencer-buchness.css" href="spencer-buchness.css">
```
Comment (tkraak) [inline spencer-buchness.html:5]: there is no such "type" :)

File: spencer-buchness.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+</head>
+
+<body>
+<h1>Brief History of CSS</h1>
+
+<ul>
+<li><div id="happy-cake"><aside class="gnarly">1994:</aside> The first work on style sheets started at CERN.<div></li>
```
Comment (tkraak) [inline spencer-buchness.html:12]: `<aside>` is semantically incorrect here

File: spencer-buchness.html (hunk 1 above)
Comment (SBuchness) [inline spencer-buchness.html:5]: Haha crap. I googled how to call in css and it had those two commands which
I thought were weird but it seemed to work for me. I'll fix it

File: spencer-buchness.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+.gnarly {
+  font-weight: bold;
+  font-style: italic;
+}
+
+li > a {
+  color: pink;
+}
+
+blockquote {
```
Comment (tkraak) [inline spencer-buchness.css:18]: that's not a descendant selector

**Pull request closed without merging**